packages/application-config/package.json

```diff
@@ -0,0 +1,85 @@
+{
+  "name": "@commercetools-frontend/application-config",
+  "version": "1.0.0",
+  "description": "Configuration utilities for building Custom Applications",
+  "bugs": "https://github.com/commercetools/merchant-center-application-kit/issues",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/commercetools/merchant-center-application-kit.git",
+    "directory": "packages/application-config"
+  },
+  "homepage": "https://docs.commercetools.com/custom-applications",
+  "keywords": [
+    "javascript",
+    "frontend",
+    "react",
+    "toolkit",
+    "config"
+  ],
+  "license": "MIT",
+  "private": true,
+  "publishConfig": {
+    "access": "public"
+  },
+  "main": "./build/index.js",
+  "module": "./build/index.js",
+  "typings": "./build/typings/src/index.d.ts",
+  "types": "./build/typings/src/index.d.ts",
+  "files": [
+    "build",
+    "schema.json",
+    "package.json",
+    "LICENSE",
+    "README.md"
+  ],
+  "scripts": {
+    "prepare": "./../../scripts/version.js replace",
+    "prebuild": "rimraf build/**",
+    "build": "yarn build:schema && yarn build:src && yarn build:typings",
+    "build:schema": "json2ts schema.json src/schema.ts --style.singleQuote --bannerComment '/* eslint-disable prettier/prettier */\n// This file was automatically generated by json-schema-to-typescript.\n// DO NOT MODIFY IT BY HAND. Instead, modify the source schema.json file.'",
+    "build:src": "babel src --out-dir build --extensions .ts --ignore src/types.ts,src/schema.ts",
+    "build:typings": "cross-env tsc -p tsconfig.declarations.json --emitDeclarationOnly --declarationDir build/typings"
+  },
+  "dependencies": {
+    "@babel/runtime": "7.10.4",
+    "@babel/runtime-corejs3": "7.10.4",
+    "ajv": "6.12.3",
+    "core-js": "3.6.5",
+    "cosmiconfig": "6.0.0",
+    "lodash": "4.17.19"
+  },
+  "devDependencies": {
+    "@babel/plugin-transform-runtime": "7.10.4",
+    "@babel/preset-env": "7.10.4",
+    "@babel/preset-typescript": "7.10.4",
+    "json-schema-to-typescript": "9.1.1"
+  },
+  "engines": {
+    "node": ">=10",
+    "npm": ">=5"
+  },
+  "babel": {
```

Comment:
Mostly to avoid confusing tools, as this config is only used by this package for development. We do the same in other packages (mc-html-template, mc-scripts)